qutip/solver/floquet.py

```diff
@@ -496,7 +527,15 @@ def floquet_tensor(H, c_ops, spectra_cb, T=0, w_th=0.0, kmax=5, nT=100):
     return Qobj(r, dims=[dims, dims], superrep="super", copy=False)
 
 
-def fsesolve(H, psi0, tlist, e_ops=None, T=0.0, args=None, options=None):
+def fsesolve(
+    H: QobjEvoLike,
```

Comment:
There might be conversion `QobjEvoLike` -> `QobjEvo` missing in the function (and also `fmesolve`)